src/neuroconv/datainterfaces/ophys/scanimage/scanimageimaginginterface.py

```diff
@@ -57,6 +58,7 @@ def __init__(
             assert fallback_sampling_frequency is not None, assert_msg
             sampling_frequency = fallback_sampling_frequency
 
+        self.keyword_aliases = dict(fallback_sampling_frequency="sampling_frequency")
```

Comment:
This is what I could come up with when the input arguments to an interface don't 100% correspond to the way we initialize them with SI and store the record as `self.source_data`

Reply:
Actually it's not going to work in more complicated cases like SpikeGLX where we take a `file_path` and alias it to a `folder_path` - thinking on a better solution